google/cloud/aiplatform/training_jobs.py

```diff
@@ -17,20 +17,67 @@
 
 import datetime
 import functools
+import logging
 import pathlib
 import shutil
 import subprocess
 import sys
 import tempfile
-from typing import Callable, Dict, Optional, Sequence
+import time
+from typing import Callable, Dict, List, Optional, Sequence
 
 
 from google.auth import credentials as auth_credentials
 from google.cloud.aiplatform import base
 from google.cloud.aiplatform import datasets
+from google.cloud.aiplatform import initializer
 from google.cloud.aiplatform import models
```

Comment:
Do we have a general guideline for when to import modules (e.g. `from google.cloud.aiplatform import models` here) versus when to import classes (e.g. `from google.cloud.aiplatform_v1beta1 import Model` below)?  

if there are simple rules of deciding this, please add to the contributing guide file for reference.

Reply:
If there is no standard defined then we should go with the go/pystyle#imports.

I'll make changes in this PR and we should ensure future PRs follow that style.

I also have a preference to reduce levels of indirection of imports as well so we should be importing directly from the module that contains the implementation.

@vinnysenthil 
@sirtorry 
@ivanmkc